libs/shared/src/services/http-client/api.client.ts

```diff
@@ -14,47 +12,89 @@ export interface IPaginatedResponse<T = unknown> {
 }
 
 export class HttpClient {
-  private axiosClient: AxiosInstance;
+  private backendUrl: string;
+  private apiVersion = 'v1';
```

Comment:
Should we make this a public configurable option field to support future versions as well?